eng/Version.Details.xml

```diff
@@ -1,131 +1,131 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="9.0.100-preview.5.24229.2" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
+    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="9.0.100-preview.5.24253.16">
       <Uri>https://github.com/dotnet/installer</Uri>
-      <Sha>d301a122c4f9d017cdc3a506f831e2edfc124ea7</Sha>
+      <Sha>fa261b952d702c6bd604728fcbdb58ac071a22b1</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="9.0.0-preview.4.24223.11" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
+    <Dependency Name="Microsoft.NETCore.App.Ref" Version="9.0.0-preview.5.24264.1">
```

Comment:
These are newer than what we have (our subscription is weekly):
* https://github.com/xamarin/xamarin-android/blob/main/eng/Version.Details.xml

So, I'd only use this version if you're willing to troubleshoot ahead of the Android workload.

Reply:
The other bumps seem stuck with some issue or other, so this is tentative to have a coherent bump for preview5.